react/features/base/redux/functions.js

```diff
@@ -19,7 +19,7 @@ export function assign(target: Object, source: Object) {
     let t = target;
 
     for (const property in source) { // eslint-disable-line guard-for-in
-        t = set(t, property, source[property], t === target);
+        t = set(t, property, source[property]);
```

Comment:
@paweldomas & @saghul, are you sure that's the correct fix? I think I meant to use `_set` here. The intention of the code is (1) to return `target` if none of the setters lead to an actual change, (2) to not modify `target`, and (3) to do it with a 1 `copyOnWrite` at most.

Reply:
@lyubomir That sounds about right. I'll make the change and push that commit separately.